src/domain/chainContext.ts

```diff
@@ -225,19 +225,19 @@ export class ChainContext {
     }
 
     // Otherwise, run the block watcher
-    return await this.runBlockWatcher();
+    return await this.runBlockWatcher(watchdogTimeout);
   }
 
   /**
    * Run the block watcher for the chain. As new blocks come in:
    * 1. Check if there are any `ConditionalOrderCreated` events, and index these.
    * 2. Check if any orders want to create discrete orders.
    */
-  private async runBlockWatcher() {
+  private async runBlockWatcher(watchdogTimeout: number) {
     const { provider, registry, chainId } = this;
     const log = getLogger(`chainContext:runBlockWatcher:${chainId}`);
     // Watch for new blocks
-    log.info("👀 Start block watcher");
+    log.info(`👀 Start block watcher with ${watchdogTimeout}s timeout`);
```

Comment:
ubernit: I think is not super relevant the timeout in the info message. Could be DEBUG. We don't expose any of the configs. If you find it useful, we can expose the relevant ones under DEBUG

Reply:
Solid standard for not emitting config as a base rule for >= INFO. Configured to emit this for `debug` level in a separate line.